BlockchainSdk/BlockchainService/EtheremJsonRpсProvider/EthereumJsonRpcList.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  EthereumJsonRpcList.swift
+//  BlockchainSdk
+//
+//  Created by skibinalexander on 25.07.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+/*
+ Use this list only for EVM-like blockchains
+ Support link - https://chainlist.org
+*/
+
+struct EthereumJsonRpcList {
+    // MARK: - Private Properties
+    
+    private let infuraProjectId: String
+    private let nowNodesApiKey: String
+    private let getBlockApiKey: String
+    private let quickNodeBscCredentials: BlockchainSdkConfig.QuickNodeCredentials
+    
+    // MARK: - Init
+    init(apiKeys: EthereumApiKeys) {
+        self.infuraProjectId = apiKeys.infuraProjectId
+        self.nowNodesApiKey = apiKeys.nowNodesApiKey
+        self.getBlockApiKey = apiKeys.getBlockApiKey
+        self.quickNodeBscCredentials = apiKeys.quickNodeBscCredentials
+    }
+    
+    // MARK: - Implementation
+    
+    public func getJsonRpc(for blockchain: Blockchain) -> [URL] {
+        switch blockchain {
```

Comment:
```suggestion
    public func getJsonRpc(for blockchain: Blockchain) -> [URL] {
              jsonRpc(for: blockchain).compactMap { URL(string: $0) }
        }
```

Может сделать внизу приватные метод jsonRpc(или по другому назвать) 
и там возвращать просто массив [String]
а что сейчас очень много раз написано URL(string:)
Как будто должно быть почище